spec/index.html

```diff
@@ -1125,18 +1125,27 @@ <h3>Reifying Statements: <code>rdf:ID</code></h3>
   </dl>
 
   <section id="section-MIME-Type">
-    <h2>RDF MIME Type, File Extension and Macintosh File Type</h2>
+    <h2>RDF/XML Internet Media Type, File Extension and Macintosh File Type</h2>
+    <dl>
+      <dt>Contact:</dt>
+      <dd>Dan Brickley</dd>
+      <dt>See also:</dt>
 
-    <p>The Internet media type / MIME type for RDF/XML is
-    <code>application/rdf+xml</code> &mdash; [[RFC3870]].
-    </p>
+      <dd><a href="http://www.w3.org/2002/06/registering-mediatype">How to Register a Media Type for a W3C Specification</a></dd>
+      <dd><a href="http://www.w3.org/2001/tag/2002/0129-mime">Internet Media Type registration, consistency of use</a><br />TAG Finding 3 June 2002 (Revised 4 September 2002)</dd>
+    </dl>
 
-    <p>It is recommended that RDF/XML files have the extension
-    <code>".rdf"</code> (all lowercase) on all platforms.</p>
+    <p>The Internet Media Type / MIME type for RDF/XML is
+      <code>application/rdf+xml</code> &mdash; [[RFC3870]].</p>
+
+    <p>It is recommended that RDF/XML files have the extension &quot;.rdf&quot;
+      (all lowercase) on all platforms.</p>
 
     <p>It is recommended that RDF/XML files stored on Macintosh HFS file
-    systems be given a file type of <code>"rdf&nbsp;"</code>
-    (all lowercase, with a space character as the fourth letter).</p>
+      systems be given a file type of <code>"rdf&nbsp;"</code>
+      (all lowercase, with a space character as the fourth letter).</p>
```

Comment:
As with `.rdf`, I suggest departing from `rdf ` and instead specifying `rdfx` as the file type for macOS. That said, macOS has long since left this style of file type handling behind, and most reasonably current systems won't care about this, especially if the files are named with the `.rdfxml` extension, which users can manually associate with their preferred RDF/XML editor, in the unlikely event that such is needed.
```suggestion
      systems be given a file type of <code>"rdfx"</code>
      (all lowercase).</p>
```

Reply:
This would make an incompatibility with all existing RDF/XML files that might already use this convention thus potentially making this an incompatible chang.e

Reply:
We should not change this.
It is in effect adding a extension, because so many file exist that already use `.rdf` and will continue to.

Reply:
> It is in effect adding a extension, because so many file exist that already use `.rdf`
 and will continue to.

This entirely misunderstands the `rdf ` tile type of the Classic Mac OS (9 and previous). There is no file extension here. I would also suggest that there are relatively few RDF/XML files on the Classic Mac OS, as it was EOLed very near to RDF 1.0's debut.

Reply:
I missed placed my comment. I will move it.